app.js

```diff
@@ -60,7 +68,17 @@ app.controller('Damage', function ($scope) {
     }
   };
 
-  $scope.hits = [100, 500, 1000, 2000, 3000, 4000, 5000, 7500, 10000];
+  if ($scope.damageDefaults === null) {
+    $scope.damageDefaults = [100, 500, 1000, 2000, 3000, 4000, 5000, 7500, 10000, null, null];
+  }
+  $scope.hits = [];
+  for (var i = 0; i < $scope.damageDefaults.length; i++) {
+    $scope.hits.push({
+      damage: $scope.damageDefaults[i],
+      id: "hit" + i
```

Comment:
If we decide to store only the modified ones or something I thought we'd use them.